_posts/2019-10-09-hhvm-4.26.0.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "HHVM 4.26.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.26 is released! This release marks the end of support for 4.20;
+4.21&ndash;4.25 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- various IDE integration improvements (better autocompletion, more contextual information on hover)
+- values with [`dynamic` type](https://docs.hhvm.com/hack/built-in-types/dynamic) can now be used in more places (inside XHP, `unset` statements)
+- it is no longer a Hack error to declare a method with the same name as its encompassing class (previously error 3007)
```

Comment:
Just to check: these do not act as constructors any more, right?

Probably worth noting that these are just standard methods now.

Reply:
Sure, probably worth mentioning. But they haven't been constructors for a long time (any 4.x version I tried).

They work as constructors in 3.30 with a warning.

```
$ hhvm cl-meth-name.hack 

Deprecated: Methods with the same name as their class will not be constructors in a future version of PHP; ClMethName has a deprecated constructor in /home/jjergus/work/code/hack-examples/cl-meth-name.hack on line 3
method called
```